delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/config/ConfigReaderInstances.scala

```diff
@@ -77,6 +80,18 @@ trait ConfigReaderInstances {
     }
   }
 
+  implicit final val indexingConfigReader: ConfigReader[IndexingConfig] = {
+    val logger: Logger = Logger[IndexingConfig]
+
+    @nowarn("cat=unused")
+    implicit val retryStrategyConfig: ConfigReader[RetryStrategy] =
```

Comment:
I think the `IndexingConfig` should only refer to `RetryStrategyConfig`, not an actual retry strategy. The retry strategy only makes sense within that module, based on a config.